plugins/modules/btrfs_info.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python
+
+# Copyright (c) 2022, Gregory Furlong <gnfzdz@fzdz.io>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: btrfs_info
+short_description: Returns the current status of all identified btrfs filesystems
+version_added: "1.0.0"
```

Comment:
The next release will be 6.3.0. 1.0.0 has already been years ago :)
```suggestion
version_added: "6.3.0"
```

Reply:
Since this sat for a bit longer than intended, Ive updated again to 6.6.0. Let me know if that's incorrect.

Reply:
6.6.0 is the next release, so :+1: for now ;)